website/static/js/filepage/revisions.js

```diff
@@ -97,12 +105,18 @@ var FileRevisionsTable = {
                     model.hasDate ? m('th', 'Date') : false,
                     model.hasUser ? m('th', 'User') : false,
                     m('th[colspan=2]', 'Download'),
-                    model.hasHashes ? m('th', 'MD5') : false,
-                    model.hasHashes ? m('th', 'SHA2') : false,
+                    model.hasHashes ? m('th', [
+                        'MD5 ', m('#md5pop.fa.fa-question-circle[data-content="Unique 32-digit code for the file."][rel="popover"]' +
```

Comment:
It's best to avoid reliance on CSS IDs, as they are brittle (changing the ID will break the code). If you do the refactor suggested in https://github.com/CenterForOpenScience/osf.io/pull/5148/files#r55424839 , you should be able to remove the IDs here.